lib/ios.js

```diff
@@ -226,9 +226,27 @@ class OptionalIOSWebkitDebugProxyCommandCheck extends DoctorCheck {
 }
 checks.push(new OptionalIOSWebkitDebugProxyCommandCheck());
 
+class OptionalIfuseCommandCheck extends DoctorCheck {
```

Comment:
not sure if it makes sense to only enable this verification for osx

Reply:
Do you mean we should skip ios related checking on non-osx env?
I agree with it. let me create another pr.